package_control/package_manager.py

```diff
@@ -1393,6 +1397,11 @@ def add_extracted_dirs(dir_):
                             ''',
                             package_name
                         )
+                    finally:
```

Comment:
I would expect this to not happen after an exception was thrown.
```suggestion
                    else:
```